artemis-server/src/test/resources/ConfigurationTest-full-config.xml

```diff
@@ -158,6 +158,14 @@
             <target-key-filter>^[^.]+</target-key-filter>
             <local-target-filter>DEFAULT</local-target-filter>
          </broker-balancer>
+         <broker-balancer name="simple-local-with-transformer">
+            <target-key>CLIENT_ID</target-key>
+            <target-key-filter>^[^.]+</target-key-filter>
+            <local-target-filter>DEFAULT</local-target-filter>
+            <key-transformer name="CONSISTENT_HASH_MODULO">
```

Comment:
@brusdev I dropped the target- prefix on key for key-transformer.

I am thinking a separate PR to maybe remove target- from target-key and target-key-filter, or a general revisit on names. A key, could really be a key-type etc... naming is very hard but I think that is a question of a separate PR.

If you like I could leave the name target-key-transformer - but in that case I may want to call it local-target-key-transformer - as that is more exact.

Reply:
maybe is should be local-target-key-transformer, where it is a local-target and not a target-key, ie: removing target from key will still make sense.